spec/controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rack/test'
+require 'controller'
+
+describe Controller do
+    controller = Controller.new
+
+  it "creates an new game" do
+    expect(controller.game.board.cells.size).to eq 9
```

Comment:
think about law of demeter. Should the controller care how many cells the board has? What is the responsibility of the controller, and what behaviour do you really want to check?